modules/installation-creating-azure-worker.adoc

```diff
@@ -2,23 +2,17 @@
 //
 // * installing/installing_azure/installing-azure-user-infra.adoc
 // * installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc
-// * installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc
 
 ifeval::["{context}" == "installing-azure-user-infra"]
 :azure:
 :cp: Azure
-:three-node-cluster:
 endif::[]
 ifeval::["{context}" == "installing-azure-stack-hub-user-infra"]
 :ash:
 :cp: Azure Stack Hub
 endif::[]
-ifeval::["{context}" == "installing-restricted-networks-azure-user-provisioned"]
-:azure:
-:cp: Azure
-endif::[]
 
-:_mod-docs-content-type: PROCEDURE
+:_content-type: PROCEDURE
```

Comment:
The meta data should not be updated. `_mod-docs-content-type: PROCEDURE` is the correct meta data.